docs/config.md

```diff
@@ -244,6 +244,9 @@ profile_models: <comma-delimited-string-list|list|profile_model>
 # List of objectives that user wants to sort the results by it.
 [ objectives: <objective|list> ]
 
+# Weighting used to bias the model's objectives (agianst the other models) in multi-mode mode
```

Comment:
My only comment is that we need to somehow communicate (here and/or in your other section) that this is not a global option, and must be specified down at the model level inside of profile_models

Reply:
Do we? There's no example that uses it globally. It's completely non-sensical to have a single number in the global option. And, if a user does try it that get a very nice and descriptive error message telling them that it can't be global.